pkg/kubecost/set.go

```diff
@@ -0,0 +1,13 @@
+package kubecost
+
+import "encoding"
+
+type Set interface {
+	CloneSet() Set
```

Comment:
Just `Clone` here

Reply:
The problem with Clone is that it that the signature is already taken all over the code base with the incorrect return